src/redismodule.h

```diff
@@ -860,7 +861,7 @@ typedef struct RedisModuleTypeMethods {
 } RedisModuleTypeMethods;
 
 #define REDISMODULE_GET_API(name) \
-    RedisModule_GetApi("RedisModule_" #name, ((void **)&RedisModule_ ## name))
+    assert(RedisModule_GetApi("RedisModule_" #name, ((void **)&RedisModule_ ## name)) == REDISMODULE_OK)
```

Comment:
for a moment i thought that we might have issues with NDEBUG if the caller defined it.
but these lines are actually compiled as part of redis binary.
however, maybe we still rather just set a boolean and handle it at the end of RedisModule_Init?
arguably a static assertion would have been sufficient if we had such an option, and i imagine that adding an assertion per line would blow up the executable size a bit...

Reply:
i ended up using a static REDISMODULE_GET_API_RET, and using `RedisModule_Assert` handle it at the end of RedisModule_Init. using `RedisModule_Assert` so that we can print out the error log